moveit_core/robot_state/include/moveit/robot_state/cartesian_interpolator.h

```diff
@@ -144,7 +144,7 @@ class CartesianInterpolator
 
      The Cartesian path to be followed is specified as a \e translation vector to be followed by the robot \e link.
      This vector is assumed to be specified either in the global reference frame or in the local
-     reference frame of the link (\e global_reference_frame is false).
```

Comment:
I know this is just a port of the previous PR, but why was this comment removed? (@rhaschke) The `global_reference_frame` arg is still present.

Reply:
Added the comment back